include/hlsl++/common.h

```diff
@@ -248,7 +248,9 @@ namespace hlslpp
 #endif
 
 	// Reference http://www.liranuna.com/sse-intrinsics-optimizations-in-popular-compilers/
+#if !defined(_hlslpp_sign_ps)
```

Comment:
Hoping to use this pattern to allow for specific implementations to provide possible "overrides" for these higher-level functions in the case of additional feature-sets.
Ex, AVX512 has a single-instruction form of `_hlslpp_countbits_epu32` and may define `_hlslpp_countbits_epu32` as a preprocessor to override the default `_hlslpp_countbits_epu32` in `vector_uint.h`.

[Same for `_hlslpp_reversebits_epu32` in the case of GFNI and possibly others](https://wunkolo.github.io/post/2020/11/gf2p8affineqb-bit-reversal/).